tools/db_bench_tool.cc

```diff
@@ -5407,7 +5407,7 @@ class Benchmark {
         }
         if (levelMeta.level == 0) {
           for (auto& fileMeta : levelMeta.files) {
-            fprintf(stdout, "Level[%d]: %s(size: %" ROCKSDB_PRIszt " bytes)\n",
+            fprintf(stdout, "Level[%d]: %s(size: %" PRIi64 " bytes)\n",
```

Comment:
Shouldn't this be PRIu64?

Reply:
Yes, I was just copying from a few lines down. Doesn't matter in practice.